app/uk/gov/hmrc/eoricommoncomponent/frontend/connector/TaxEnrolmentsConnector.scala

```diff
@@ -81,30 +77,7 @@ class TaxEnrolmentsConnector @Inject() (http: HttpClient, appConfig: AppConfig,
     hc: HeaderCarrier
   ): Future[HttpResponse] = {
     val url = s"$baseUrl/$serviceContext/service/$enrolmentKey/enrolment"
-    http.PUT[GovernmentGatewayEnrolmentRequest, HttpResponse](url = url, body = request).map { response =>
-      auditCall(url, request, response)
-      response
-    }
+    http.PUT[GovernmentGatewayEnrolmentRequest, HttpResponse](url = url, body = request)
   }
 
-  private def auditCall(url: String, request: TaxEnrolmentsRequest, response: HttpResponse)(implicit
-    hc: HeaderCarrier
-  ): Unit =
-    audit.sendExtendedDataEvent(
-      transactionName = "Issuer-Call",
-      path = url,
-      details = Json.toJson(RequestResponse(Json.toJson(request), Json.toJson(response.status))),
-      eventType = "IssuerCall"
-    )
-
-  private def auditCall(url: String, request: GovernmentGatewayEnrolmentRequest, response: HttpResponse)(implicit
-    hc: HeaderCarrier
-  ): Unit =
-    audit.sendExtendedDataEvent(
-      transactionName = "Synchronous-Enrolment",
-      path = url,
-      details = Json.toJson(RequestResponse(Json.toJson(request), Json.toJson(response.status))),
-      eventType = "SynchronousEnrolment"
-    )
```

Comment:
I removed them, because they had only request with status response. This can be covered by implicit audit events IMO.